client/js/middleware/serialization/qbank/serializers/survey.jsx

```diff
@@ -0,0 +1,78 @@
+import _              from 'lodash';
+import baseSerializer from './base';
+import { scrub }      from '../../serializer_utils';
+import genusTypes     from '../../../../constants/genus_types';
+import guid           from '../../../../utils/guid';
+
+function serializeChoices(originalChoices, newChoiceAttributes) {
+  const choices = _.map(originalChoices, (choice) => {
+    const updateValues = newChoiceAttributes[choice.id];
+    return {
+      id: choice.id,
+      text: _.get(updateValues, 'text') || choice.text,
```

Comment:
Oh cool, I didn't realize it worked if the first parameter is null. Is it worth refactoring to use `.get(updateValues, 'text', choice.text)` to make use of the default value parameter